### PR TITLE
Upgdate to latest wgpu-native

### PR DESCRIPTION
**Pull request by rukai**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ impl CommandEncoder {
 
 impl<'a> RenderPass<'a> {
     /// Sets the active bind group for a given bind group index.
-    pub fn set_bind_group(&mut self, index: u32, bind_group: &BindGroup, offsets: &[u32]) {
+    pub fn set_bind_group(&mut self, index: u32, bind_group: &BindGroup, offsets: &[u64]) {
```
Comment (kvark) [inline src/lib.rs]: Can we use the `BufferAddress` type for the offsets? It's supposed to be imported from `wgpu-native`

**Comment by rukai** [comment]
Updated to latest commit, should be good to merge.

**Comment by kvark** [comment]
Bors r+

**Comment by m4b** [comment]
I will test this tonight and let you all know :)

**Comment by m4b** [comment]
Woohoo, yup, 84a9f0b8f442e027b909b120c8d2b43064b708c2 no longer has the image validation error :) great work!

**Comment by kvark** [comment]
Awesome, thanks for confirmation!

**Pull request merged**